x-pack/plugin/snapshot-based-recoveries/src/internalClusterTest/java/org/elasticsearch/xpack/snapshotbasedrecoveries/recovery/SnapshotBasedIndexRecoveryIT.java

```diff
@@ -673,7 +678,11 @@ public void testCancelledRecoveryAbortsDownloadPromptly() throws Exception {
             String indexName = randomAlphaOfLength(10).toLowerCase(Locale.ROOT);
             createIndex(
                 indexName,
-                Settings.builder().put(IndexMetadata.SETTING_NUMBER_OF_SHARDS, 1).put(IndexMetadata.SETTING_NUMBER_OF_REPLICAS, 0).build()
+                Settings.builder()
+                    .put(IndexMetadata.SETTING_NUMBER_OF_SHARDS, 1)
+                    .put(IndexMetadata.SETTING_NUMBER_OF_REPLICAS, 0)
+                    .put(MergePolicyConfig.INDEX_MERGE_ENABLED, false)
```

Comment:
Can you add a comment with the reason why we are disabling merges?

Reply:
Sure, see 0edf05ecbaedfb9aa3a16da31220028a9d201686 (although it's the same reason we disable merges in all the other tests in this suite too)

Apologies for the accidental force-push there - I force-pushed back to the original state again.